backend-services/user-verification-service/src/main/java/org/cardano/foundation/voting/service/discord/DefaultDiscordUserVerificationService.java

```diff
@@ -465,13 +471,18 @@ private Either<Problem, IsVerifiedResponse> handleKeriVerification(DiscordCheckV
 
         log.info("OOBI registered successfully: {}", oobiM);
 
-        // Step 3: Attempt to verify OOBI registration up to 10 times
+        // Step 3: Attempt to verify OOBI registration up to 60 times
         val oobiFetchResultE = keriVerificationClient.getOOBI(oobi, 60);
         if (oobiFetchResultE.isLeft()) {
             return Either.left(oobiFetchResultE.getLeft());
         }
 
-        // Step 4: Verify signature after OOBI registration
+        // Step 4: Update key state
+        Either<Problem, Boolean> keyStateUpdateResult = keriVerificationClient.updateAndVerifyKeyState(walletId, 60);
```

Comment:
We don't need to update the key state if we've just resolved the OOBI - it will have the latest key state from resolve time. The update to the key state is for key rotations thereafter